README.md

```diff
@@ -60,29 +61,32 @@ Then in `main.rs`, copy the below:
 ```rust
 use boa_engine::{Context, Source};
 
-let js_code = r#"
-    let two = 1 + 1;
-    let definitely_not_four = two + "2";
-
-    definitely_not_four
-"#;
-
-// Instantiate the execution context
-let mut context = Context::default();
-
-// Parse the source code
-match context.eval(Source::from_bytes(js_code)) {
-    Ok(res) => {
-        println!(
-            "{}",
-            res.to_string(&mut context).unwrap().to_std_string_escaped()
-        );
-    }
-    Err(e) => {
-        // Pretty print the error
-        eprintln!("Uncaught {e}");
-    }
-};
+fn main() {
+  let js_code = r#"
+      let two = 1 + 1;
+      let definitely_not_four = two + "2";
+
+      definitely_not_four
+  "#;
+
+  // Instantiate the execution context
+  let mut context = Context::default();
+
+  // Parse the source code
+  match context.eval(Source::from_bytes(js_code)) {
+      Ok(res) => {
+          println!(
+              "{}",
+              res.to_string(&mut context).unwrap().to_std_string_escaped()
```

Comment:
Since it's wrapped in a function, I Would prefer using the `?` error propagation and making main return a JsResult.

If not then using a display method that does not return a result `res.display()` we can also remove the `.to_std_string_escaped()` since it implements `Display`